python/tvm/topi/x86/dense_alter_op.py

```diff
@@ -36,6 +36,11 @@ def _alter_dense_layout(attrs, inputs, tinfos, out_type):
         and tinfos[1].dtype == "float32"
         and out_type.dtype == "float32"
     ):
+        # mlas is only used for static tensors
+        if any([isinstance(dim, tvm.tir.Any) for dim in tinfos[0].shape]) or any(
+            [isinstance(dim, tvm.tir.Any) for dim in tinfos[1].shape]
+        ):
+            return None
```

Comment:
Do we want to return None here or continue to code after this?

Reply:
Thanks, good catch!

Reply:
Updated.